zerver/lib/home.py

```diff
@@ -239,4 +240,10 @@ def build_page_params_for_home_page_load(
 
     page_params["translation_data"] = get_language_translation_data(request_language)
 
+    if user_profile is None:
+        # Get rendered version of realm description which is displayed in right
+        # sidebar for spectator.
+        # TODO: Remove once right sidebar is hidden for spectators.
```

Comment:
I think we can remove this TODO; there's a good chance we'll offer the realm description somewhere else in the spectator UI.